Mono.Cecil.Cil/Instruction.cs

```diff
@@ -122,11 +123,11 @@ public override string ToString ()
 				break;
 			case OperandType.InlineString:
 				instruction.Append ('\"');
-				instruction.Append (operand);
+				instruction.Append (Convert.ToString(operand, CultureInfo.InvariantCulture));
```

Comment:
This is just a string, right? No need to convert a string into the invariant culture.

Reply:
Did not notice that. Thanks!